projects/storefrontapp-e2e-cypress/cypress/helpers/register.ts

```diff
@@ -3,7 +3,7 @@ import { register } from './auth-forms';
 export const loginLink = 'cx-login [role="link"]';
 
 export function registerUser(user) {
-  cy.get(loginLink).click();
+  cy.getByText(/Sign in \/ Register/i).click();
```

Comment:
I have no doubt that this works but couldn't there be an issue where this is not fully loaded yet so it's not clickable? Basically this won't fix that.

Reply:
The issue here is that the test is targeting the selector cx-login [role="link"].
This selector exists both when the user is logged in and when they're not logged in.
Every nav in the My-Account dropdown contains the attribute role="link"

So cypress logs out and then immediately tries to log back in since the selector is still there. But, since it does it so fast it's actually looking at "My Account" and therefore finds several options to click between. My solution was to not look at the selector since it persists but rather to look and see if the Sign/Register text is there.

I should have included the context when submitting the PR, sorry about that.

Reply:
Ok this sounds good 😉